docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -334,6 +334,12 @@ export access_token=$(\
  )
 ----
 
+[NOTE]
+====
+If the setting below is set, you should add the openid to the scope in the query parameters.
```

Comment:
Which `setting below`? I wasn't able to figure out what this note was about.

Also maybe `user info` would be better than `openid`?

Reply:
Right, I agree, this is why I proposed a rewording, see my suggestion above